main.go

```diff
@@ -12,6 +12,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/ayanamist/gdns-go/myip"
```

Comment:
AFAIK, relative import are discouraged, but I can't seem to find the document that support it. I'm leaning toward using absolute import path everywhere.